crates/bin/node/src/grpc_service.rs

```diff
@@ -402,4 +404,48 @@ impl Node for GrpcState {
             expiry: response.expiry,
         }))
     }
+
+    async fn get_node_info(
+        &self,
+        _node_info_request: Request<GetNodeInfoRequest>,
+    ) -> Result<Response<NodeInfoResponse>, Status> {
+        let nuts_config = {
+            let nuts_read_lock = self.nuts.read().await;
+            nuts_read_lock.clone()
+        };
+        let pub_key = {
+            let mut signer_write_lock = self.signer.write().await;
+            signer_write_lock
+                .get_root_pub_key(Request::new(GetRootPubKeyRequest {}))
+                .await?
+                .into_inner()
+                .root_pubkey
+        };
+        let node_info = NodeInfo {
+            name: Some("Paynet Test Node".to_string()),
+            pubkey: Some(PublicKey::from_str(&pub_key).unwrap()),
+            version: Some(NodeVersion {
+                name: "some_name".to_string(),
+                version: "0.0.0".to_string(),
+            }),
+            description: Some("A test node".to_string()),
+            description_long: Some("This is a longer description of the test node.".to_string()),
+            contact: Some(vec![ContactInfo {
+                method: "some_method".to_string(),
+                info: "some_info".to_string(),
+            }]),
+            nuts: nuts_config,
+            icon_url: Some("http://example.com/icon.png".to_string()),
+            urls: Some(vec!["http://example.com".to_string()]),
+            motd: Some("Welcome to the node!".to_string()),
+            time: Some(1616161616),
```

Comment:
read the system time instead
try this: `std::time::UNIX_EPOCH.elapsed().unwrap()`